README.md

```diff
@@ -1,3 +1,11 @@
 # GTFS to GeoJson
 
 This Rust crate is used to convert GTFS data to the GeoJSON format.
+
+## How to compile and run the standalone program
+
+* Clone this repository
+* [Install Rust](https://www.rust-lang.org/tools/install)
+* Run the tests with `cargo test --all-features` (see [GitHub actions setup](https://github.com/rust-transit/gtfs-to-geojson/tree/main/.github/workflows))
+* Build the optimized binary with `cargo build --release`
+* Run the standalone program with `target/release/gtfs-geojson --help`
```

Comment:
maybe we can put first that the build/run steps can be run with `cargo run --release -- --help`?

Reply:
I have added that (I am keeping the "non cargo" version because this is what I will use when shipping the binary somewhere). Thanks!